guides/common/assembly_synchronizing-content-between-servers.adoc

```diff
@@ -1,33 +1,31 @@
 include::modules/con_synchronizing-content-between-servers.adoc[]
 
-include::modules/con_exporting-from-a-connected-server.adoc[leveloffset=+1]
+include::modules/con_how-to-synchronize-content-using-export-and-import.adoc[leveloffset=+1]
```

Comment:
I wonder which assembly  doc has the [guides/common/modules/con_how-to-use-iss-based-on-your-network-setup.adoc](https://github.com/theforeman/foreman-documentation/pull/1290/files#diff-2a0fe7541505e3eb05f21f810a9323afc357aa863c3ab244a3a2bff73ee03fe8)

Reply:
Try to build version for Katello and revisit the introduction of the chapter. Then it should make sense.

Reply:
cc @jeremylenz ^^

Reply:
I'll add this info into the PR intro next time! :D

Reply:
Still not able to find it but will ok the content if you are sure :)